.eslintrc.json

```diff
@@ -14,5 +14,18 @@
     "node/no-missing-import": "off",
     "no-warning-comments": "off",
     "camelcase": "off"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["*.isomorphic.ts"],
+      "rules": {
+        "no-restricted-imports": ["error", {
+          "patterns": [{
+            "group": ["node:*", "!node:path"],
+            "message": "See https://github.com/knocklabs/knock-cli/pull/257 for more details."
+          }]
+        }]
+      }
+    }
+  ]
```

Comment:
@meryldakin @connorlindsey Btw I realized our README gets auto generated when we build a release so I'm not sure if that's going to work. However, I did figure out how to add this eslint rule to automatically check for any disallowed node module imports in isomorephic files, so I think we will have a good guardrail for this that way. And finally I added notes at the top of each isomorphic files for a bit more context.